standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/Msck.java

```diff
@@ -399,7 +399,7 @@ private void validateAndAddMaxTxnIdAndWriteId(long maxWriteIdOnFilesystem, long
     long maxAllocatedWriteId = getMsc().getMaxAllocatedWriteId(dbName, tableName);
     if (maxAllocatedWriteId > 0 && maxWriteIdOnFilesystem > maxAllocatedWriteId) {
       throw new MetaException(MessageFormat
-          .format("The maximum writeId {} in table {} is greater than the maximum allocated in the metastore {}",
+          .format("The maximum writeId {0} in table {1} is greater than the maximum allocated in the metastore {2}",
```

Comment:
`The highest writeId [{0}] in the table '{1}' is greater than the maximum writeId allocated in the metastore [{2}]`